syntax = "proto3";


message WebsiteFetchRequest {
    string id = 1;
    string url = 2;
}

message WebsiteFetchResponse {
    string id = 1;
    bytes content = 2;
}